packages/slider/src/ux-slider.ts

```diff
@@ -119,7 +122,7 @@ export class UxSlider implements UxComponent {
 
     this.isActive = true;
     const isMouseEvent = e instanceof MouseEvent;
-    const isTouchEvent = Array.isArray(e.touches) && e.touches.length > 0;
+    const isTouchEvent = e.touches instanceof TouchList && e.touches.length > 0;
```

Comment:
Do we need the `TouchList` check? When you test this method, you would want to pass a fake object with `touches` as an array

Reply:
Sorry not sure I understand what you mean ?

Reply:
Basically I changed here the check from Array to TouchList because the `TouchEvent.touches` is defined as a `TouchList`. But to be honest I'm not sure if this is the case for every browser, is that your concern ?

Reply:
Why not using `e instanceof TouchEvent` instead?

Reply:
I think we can use duck typing here. As long as it's defined, and it has `length > 0`, we're good to go

Reply:
IMO it's bad practice to check for an objects property to be sure to have the correct typ, instead of checking the type directly.

Reply:
The pros and cons of each method here seem quite minimal. Do we have a difference of perf between a type check (`instanceof TouchEvent`) or a property check `e.touches && e.touches.length >0`.

I think in the case of `TouchEvent` that if we check for type we can assume that `e.touches.length` is indeed bigger than `0` as a touch event is fired by at least one touch ??

Reply:
Fixed here with type checking: https://github.com/aurelia/ux/pull/256/commits/d58457cf73b96f6f10ea6e364ea07570b4410c64